### ISSUE TITLE
Apparently no way to determine if there are GPU resources available for Media SDK applications

### ISSUE THREAD

**Issue by htartisan** [Intel-Media-SDK/MediaSDK#43]
Intel provides the Intel Media SDK (MFX / QuickSync) SDK, that allows applications to accelerate H.264 encoding.  (This is great!)

However, each Intel GPU provides a limited number of GPU resources (which very depending on which GPU is on the system) that are used to do this, so only a limited number of video files can be encoded in parallel.  

Unfortunately Intel does not seem to offer any way to determine how much of the GPU resources are currently being used, so there is no way to know when a server is "fully loaded".

This seems like a pretty major over-sight to me.  How do users determine if any more Media SDK applications can be run in parallel???

**Comment by avbelova** [comment]
Hi @htartisan , 
Please try out Metrics Monitor tool that's part of Media Server Studio for Linux. It's programming API to capture GPU load metrics.

Best wishes, 
Anna

**Comment by dvrogozh** [comment]
Metrics monitor just landed here on github in samples: https://github.com/Intel-Media-SDK/MediaSDK/pull/154. We will provide supporting documentation as a next step. As of now, refer to commit message (https://github.com/Intel-Media-SDK/MediaSDK/commit/634d9f89555623891591028e1f462d2ac97211cd) for the sw requirements hints.

**Comment by adydychk** [comment]
@htartisan Do you need any more response from our side?

**Comment by adydychk** [comment]
@htartisan I close the issue but feel free to re-open it or create new issue in case of another troubles.
